src/synopsys-action/tools-parameter.ts

```diff
@@ -60,6 +60,27 @@ export class SynopsysToolsParameter {
       }
     }
 
+    if (parseToBoolean(inputs.POLARIS_PRCOMMENT_ENABLED)) {
+      info('Polaris PR comment is enabled')
+      const prCommentSeverities: string[] = []
+      const inputPrCommentSeverities = inputs.POLARIS_PRCOMMENT_SEVERITIES
+      if (inputPrCommentSeverities != null && inputPrCommentSeverities.length > 0) {
+        const severityValues = inputPrCommentSeverities.split(',')
+        for (const severity of severityValues) {
+          if (severity.trim().length > 0) {
```

Comment:
Can't we replace to  _if (severity.trim())_ to this.?

Reply:
updated